engineering/Testing/QA-best-practices.md

```diff
@@ -0,0 +1,81 @@
+# QA-best-practises
```

Comment:
The guide from Futurice is good but we could carefully examine if we should take it to be our company standards. I will ask other QAs to help review the practices.

Reply:
I think it's just suggestion, we will align for each project instead of apply as the same to all.